### ISSUE TITLE
Use Agora API instead of /api/stub/projects

### ISSUE THREAD

**Issue by escottalexander** [BuidlGuidl/onchain-impact-dashboard#61]
This issue is for swapping out the any place we are calling our stubbed projects API for the Agora API.

`https://vote.optimism.io/api_v1/projects`. Reach out if you need the API key for testing but obviously we don't want it hardcoded anywhere. Also keep in mind that this endpoint returns projects that we do not need to show. The authority on whther we need to show a project or not is the mapping returned by our `api/stub/mapping` endpoint.

**Comment by swellander** [comment]
I don't think we currently use the `api/stub/projects` endpoint anywhere in our codebase. Once the project data is initially seeded into our db, I think we'll only need to hit the Agora API when we add new projects. So this ticket will have a lot of overlap with #63

**Comment by escottalexander** [comment]
Ah, yes. That makes sense. I am adding a phase 2 label to these two issues.

**Comment by swellander** [comment]
It looks like the Agora API doesn't expose a project detail `/projects/{ID}` endpoint for some reason. So when the time comes to add new projects to our db, I guess we'll need loop over every project returned from `/projects`
